src/test/java/homework_4/custom_file_reader/CustomFileReaderTest.java

```diff
@@ -3,19 +3,44 @@
 import base.UnitBase;
 import org.junit.jupiter.api.Test;
 
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Path;
+
 import static org.junit.jupiter.api.Assertions.*;
 
 class CustomFileReaderTest extends UnitBase {
 
     @Test
-    void run() {
+    void test() throws IOException {
+        CustomFileReader reader = new CustomFileReader();
+        String text = "t.e,,w\nr,,3,,,g\na,,,d...3";
```

Comment:
Your App has ability to write to file, but if i use it and write random symbols - would this test run as expected ?